### PR TITLE
feat(blobby): Adds full functionality as a fileserver over http

### PR DESCRIPTION
**Pull request by thomastaylor312**
This is currently a draft PR that adds enough functionality to consider this a Real Application™. This is all tested manually and I have two more things I want to add:

- [x] Documentation for headers and query params
- [x] Actual integration tests

The code is pretty much ready to look at though if you want to

### CODE REVIEW

File: .github/workflows/blobby.yml [hunk 1]
```diff
@@ -4,19 +4,19 @@ on:
...
-      - "actor/blobby/**"
+      - "actor/blobby/actor/**"
+      - "actor/blobby/testing/**"
     tags:
       - "blobby-v*"
   pull_request:
     branches: [main]
     paths:
-      - "actor/blobby/**"
+      - "actor/blobby/actor/**"
```
Comment (ricochet) [inline .github/workflows/blobby.yml:14]: Is there something at `actor/blobby`  root that you don't want this to trigger on?

Comment (thomastaylor312) [reply .github/workflows/blobby.yml:14]: README. Plus just being explicit :)

**Comment by LiamRandall** [comment]
I didn't pull it down and test it, but the docs are a great start!

File: .github/workflows/blobby.yml (hunk 1 above)
Comment (ricochet) [reply .github/workflows/blobby.yml:14]: Read the README just my paranoia that we'll change source at the actor root (e.g. add a Makefile) and forget about this. Minor nit, just wanted to make sure I wasn't missing a trick.

**Review by ricochet — APPROVED** [review]

**Review by jordan-rash — APPROVED** [review]
lgtm

**Pull request merged**